src/components/ColumnName/index.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+import { Cell } from 'components/Cell'
+import { Column } from 'components/Column'

```

Comment:
import {Cell, Column} from 'components'

Reply:
Refactored all folders / components / imports to follow this.